roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -28,6 +28,9 @@ class RooFuncWrapper final : public RooAbsReal {
    RooFuncWrapper(const char *name, const char *title, std::string const &funcBody, RooArgSet const &paramSet,
                   RooArgSet const &ObsSet, const RooAbsData *data = nullptr);
 
+   RooFuncWrapper(const char *name, const char *title, std::unique_ptr<RooAbsReal> &obj, RooArgSet const &paramSet,
```

Comment:
```suggestion
   RooFuncWrapper(const char *name, const char *title, RooAbsReal const& obj, RooArgSet const &paramSet,
```
It does not make sense to take `std::unique_ptr` by reference (the C++ core guidelines even list it as a code smell). You can just take the wrapped object by reference if you don't intend to transfer the ownership.